### PR TITLE
adding new EC2Fleet resource, per Nov 20 2018 update

### PR DESCRIPTION
**Pull request by axelpavageau**

### CODE REVIEW

File: troposphere/ec2.py [hunk 1]
```diff
@@ -916,3 +916,73 @@ class LaunchTemplate(AWSObject):
...
+    }
+
+
+class FleetLaunchTemplateConfigRequest(AWSProperty):
+    props = {
+        'LaunchTemplateSpecification': (
+            FleetLaunchTemplateSpecificationRequest,
+            False
+        ),
+        'Overrides': (FleetLaunchTemplateOverridesRequest, False),
```
Comment (chizou) [inline troposphere/ec2.py]: The docs say this should be a list: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-ec2fleet-fleetlaunchtemplateconfigrequest.html#cfn-ec2-ec2fleet-fleetlaunchtemplateconfigrequest-overrides

File: troposphere/ec2.py [hunk 2]
```diff
@@ -916,3 +916,73 @@ class LaunchTemplate(AWSObject):
...
+
+class EC2Fleet(AWSObject):
+    resource_type = "AWS::EC2::EC2Fleet"
+    props = {
+        'ExcessCapacityTerminationPolicy': (basestring, False),
+        'LaunchTemplateConfigs': (FleetLaunchTemplateConfigRequest, True),
+        'OnDemandOptions': (OnDemandOptionsRequest, False),
+        'ReplaceUnhealthyInstances': (boolean, False),
+        'SpotOptions': (SpotOptionsRequest, False),
+        'TagSpecifications': [(TagSpecifications), False],
```
Comment (chizou) [inline troposphere/ec2.py]: I believe the syntax here should be `'TagSpecifications': ([TagSpecifications], False),`

File: troposphere/ec2.py (hunk 1 above)
Comment (axelpavageau) [reply troposphere/ec2.py]: you're right, thanks for your review.

File: troposphere/ec2.py (hunk 2 above)
Comment (axelpavageau) [reply troposphere/ec2.py]: right as well !

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**